### ISSUE TITLE
Regex search all doesn't work

### ISSUE THREAD

**Issue by georgetye** [godotengine/godot#45074]
When I use `for match in regex.search_all(input):` I received this error message: `Identifier expected after "for".`

**Comment by VedatGunel** [comment]
`match` is a reserved keyword, just like if, else etc. So, you can't use it as an identifier.

**Comment by georgetye** [comment]
Yes, but that's what I'm trying to use it for. I'm just copying the documentation where it says:

```
var regex = RegEx.new()
regex.compile("\\S+") # Negated whitespace character class.
var results = []
for match in regex.search_all("One  Two \n\tThree"):
    results.push_back(match.get_string())
```

However, it's not working for me.

**Comment by VedatGunel** [comment]
Documentation needs to be fixed then. Just change the `match` to something else, and it should work:
```var regex = RegEx.new()
regex.compile("\\S+") # Negated whitespace character class.
var results = []
for regex_match in regex.search_all("One  Two \n\tThree"):
    results.push_back(regex_match.get_string())```

**Comment by akien-mga** [comment]
Fixed by #45075.
